src/Entities/Boat.h

```diff
@@ -13,14 +13,26 @@
 
 
 
-
+// tolua_begin
 
 class cBoat :
 	public cEntity
 {
 	typedef cEntity super;
 
 public:
+	enum eType
```

Comment:
Hm, it just occurred to me that this might be better off being called `Material` rather than `Type` - it's much more descriptive. What do you think? (besides the lot of work this makes for you :)